app/services/articles/builder.rb

```diff
@@ -10,8 +10,8 @@ def initialize(user, tag, prefill)
       @editor_version2 = @user&.editor_version == "v2"
     end
 
-    def self.call(*args)
-      new(*args).call
+    def self.call(...)
+      new(...).call
```

Comment:
This is [arguments forward ](https://blog.saeloun.com/2019/12/04/ruby-2-7-adds-new-operator-for-arguments-forwarding.html)syntax introduced in 2.7

Reply:
TIL!